modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductOptions.testcase

```diff
@@ -1817,6 +1817,136 @@ definition {
+	@priority = 5
+	test CanViewOptionsValueWithTwoSKUsLinked {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site is created") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+		}
+
+		task ("And a simple product is created") {
+			CommerceJSONProductsAPI._addCommerceProduct(
+				catalogName = "Minium",
+				productName = "Test Simple Product",
+				productType = "Simple",
+				sku = "SIM1001");
+		}
+
+		task ("And a new option is created") {
+			for (var optionNameWithKey : list "A:white,B:red") {
+				var name = StringUtil.extractFirst(${optionNameWithKey}, ":");
+				var key = StringUtil.extractLast(${optionNameWithKey}, ":");
+
+				CommerceJSONOptionsAPI._addCommerceOption(
+					fieldType = "select",
+					optionKey = ${key},
+					optionName = ${name},
+					optionValues = "value1:value1:1,value2:value2:2",
+					skuContributor = "true");
+
+				CommerceJSONOptionsAPI._assignOptionToCommerceProduct(
+					fieldType = "select",
+					optionKey = ${key},
+					optionName = ${name},
+					productName = "Test Simple Product");
+			}
+
+			for (var optionName : list "A,B") {
+				var count = 0;
+
+				CommerceProducts.openProductsAdmin();
+
+				CommerceNavigator.searchEntry(entryName = "Simple");
+
+				CommerceNavigator.gotoEntry(entryName = "Test Simple Product");
+
+				CommerceEntry.gotoMenuTab(menuTab = "Options");
+
+				CommerceProductOptions.editProductOption(
+					key_fieldType = "Select from List",
+					key_option = ${optionName},
+					key_priceType = "Static",
+					productOptionFieldType = "true",
+					productOptionPriceType = "true");
+
+				for (var skuAndDeltaPrice : list "MIN55861:60.00,MIN93027:70.00") {
+					var count = ${count} + 1;
+					var sku = StringUtil.extractFirst(${skuAndDeltaPrice}, ":");
+					var deltaPriceValue = StringUtil.extractLast(${skuAndDeltaPrice}, ":");
+
+					CommerceOptions.setDeltaPrice(
+						deltaPrice = ${deltaPriceValue},
+						key_index = "value${count}",
+						key_optionIndex = "Option Select from List",
+						productSKU = ${sku});
+
+					Click(locator1 = "Icon#CLOSE");
+
+					SelectFrameTop();
+
+					Click(
+						key_index = ${optionName},
+						locator1 = "WorkflowMetrics#SEARCH_FIELD_MAPPINGS_METRICS_INDEXES");
+
+					SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+				}
+			}
+		}
+
+		task ("And from the SKUs tab, plus button is clicked, Generate all SKUs combination button is selected") {
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = "Simple");
+
+			CommerceNavigator.gotoEntry(entryName = "Test Simple Product");
+
+			CommerceEntry.gotoMenuTab(menuTab = "SKUs");
+
+			CommerceSKUs.generateAllSkuCombinations();
+		}
+
+		task ("Then all configured data is shown correctly upon clicking different options in store front") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "web/minium/catalog");
```

Comment:
The purpose of this test is to check the SKU updates correctly on the product details page.
Another thing to check is that when we add to cart both options and SKU are correct on the minicart